Skype/SDN/articles/from-element-startorupdatetype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md

```diff
@@ -1,16 +1,17 @@
 ﻿---
-title: From element (StartOrUpdateType complexType) 
+title: From element (StartOrUpdateType complexType) (Schema C)
```

Comment:
Make title and H1 consistent.

Reply:
H1s and Title consistentency has not been flagged as an error or suggestion or warning.